src/api/procedures/createCheckpointSchedule.ts

```diff
@@ -18,8 +18,17 @@ import {
 import { filterEventRecords } from '~/utils/internal';
 
 export interface CreateCheckpointScheduleParams {
+  /**
+   * The date for when to begin creating snapshots. A null value indicates immediately
+   */
   start: Date | null;
+  /**
+   * The cadence in which to make Checkpoints. A null value indicates to create only one Checkpoint, regardless of repetitions specified
```

Comment:
I think "cadence with which to make..." is the correct usage, but I'm actually not sure and I couldn't find it anywhere, so I leave it up to you

Reply:
I think you are right.